content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -32,7 +32,7 @@ This guide describes how to use the kfctl binary to deploy Kubeflow on IBM Cloud
 
 **Note**: This section is only required when the worker nodes provider `WORKER_NODE_PROVIDER` is set to `classic`. For other infrastructures, IBM Cloud Block Storage is already set up as the cluster's default storage class.
 
-When using the `classic` worker nodes provider of IBM Cloud Kubernetes cluster, by default, it uses [IBM Cloud File Storage](https://www.ibm.com/cloud/file-storage) based on NFS as the default storage class. File Storage is designed to run RWX (read-write multiple nodes) workloads with proper security built around it. Therefore, File Storage [does not allow `fsGroup` securityContext](https://cloud.ibm.com/docs/containers?topic=containers-security#container) which is needed for DEX and Kubeflow Jupyter Server.
+When using the `classic` worker nodes provider of IBM Cloud Kubernetes cluster, by default, it uses [IBM Cloud File Storage](https://www.ibm.com/cloud/file-storage) based on NFS as the default storage class. File Storage is designed to run RWX (read-write multiple nodes) workloads with proper security built around it. Therefore, File Storage [does not allow `fsGroup` securityContext](https://cloud.ibm.com/docs/containers?topic=containers-security#container) which is needed for OIDC authservice and Kubeflow Jupyter Server.
```

Comment:
Suggestions:

- I think it's "node provider" here ("provider of nodes")
- I'd also emphasize what "it" means ("the provider"?)
- Try to talk to the user with "you" to make the language more "active" here
- "By default... default storage" - repetition
- Spell out "authservice" (OIDC) and provide a link, maybe

```suggestion
When you use use the `classic` worker node provider of IBM Cloud Kubernetes cluster, it uses [IBM Cloud File Storage](https://www.ibm.com/cloud/file-storage) based on NFS as the default storage class. File Storage is designed to run RWX (read-write multiple nodes) workloads with proper security built around it. Therefore, File Storage [does not allow `fsGroup` securityContext](https://cloud.ibm.com/docs/containers?topic=containers-security#container), which is needed for the [OIDC authentication service](https://www.kubeflow.org/docs/azure/authentication-oidc/) and Kubeflow Jupyter server.
```

Reply:
For the oidc link we should use https://github.com/arrikto/oidc-authservice. I can fix it in a follow up commit